devicon.json

```
@@ -7047,7 +7047,8 @@
         "versions": {
             "svg": [
                 "original",
-                "plain"
+                "plain",
+                "line"
             ],
             "font": [
                 "plain"
```

Add line icon to font
```suggestion
                "plain",
                "line"
```

---

Applied